.github/workflows/ruby.yml

```
@@ -0,0 +1,37 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.7']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
```

```suggestion
      uses: ruby/setup-ruby@5f29a1cd8dfebf420691c4c9a0e832e2fae5a526
```

There is a newer version of `ruby/setup-ruby` available:
https://github.com/actions/setup-ruby/releases/tag/v1.1.2

https://github.com/actions/setup-ruby/commit/5f29a1cd8dfebf420691c4c9a0e832e2fae5a526

---

Likewise here, why not just use `ruby/setup-ruby@v1`?

---

TBH, I just used the default ruby actions from the github page. Why it does not include the more general  `ruby/setup-ruby@v1` is a good question.